app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
```

Comment:
Using `self.people[name] = self` assumes that `people` is an instance attribute. However, since `people` is currently a class attribute, this line will modify the shared dictionary for all instances. Consider changing `people` to an instance attribute.